content/en/docs/ops/ambient/usage/ztunnel/index.md

```diff
@@ -91,7 +89,11 @@ caption="Basic ztunnel L4-only datapath"
 
 The figure depicts ambient pod workloads running on two nodes W1 and W2 of a Kubernetes cluster. There is a single instance of the ztunnel proxy on each node. In this scenario, application client pods C1, C2 and C3 need to access a service provided by pod S1 and there is no requirement for advanced L7 features such as L7 traffic routing or L7 traffic management so no Waypoint proxy is needed.
 
-The figure shows that pods C1 and C2 running on node W1 connect with pod S1 running on node W2 and their TCP traffic is tunneled through a single shared HBONE tunnel instance that has been created between the ztunnel proxy pods of each node. Mutual TLS (mTLS) is used for encryption as well as mutual authentication of traffic being tunneled. SPIFFE identities are used to identify the workloads on each side of the connection. The term `HBONE` (for HTTP Based Overlay Network Encapsulation) is used in Istio ambient to refer to a technique for transparently and securely tunneling TCP packets encapsulated within HTTPS packets. Some brief additional notes on HBONE are provided in a following subsection.
+The figure shows that pods C1 and C2 running on node W1 connect with pod S1 running on node W2 and their TCP traffic is tunneled through HBONE tunnel instances that have been created by the ztunnel proxy pods of each node. Mutual TLS (mTLS) is used for encryption as well as mutual authentication of traffic being tunneled. SPIFFE identities are used to identify the workloads on each side of the connection. The term `HBONE` (for HTTP Based Overlay Network Encapsulation) is used in Istio ambient to refer to a technique for transparently and securely tunneling TCP packets encapsulated within HTTPS packets. For more details on the datapath including HBONE and the traffic redirection details, refer to the accompanying document [ztunnel datapath and inpod redirection](/docs/ops/ambient/usage/inpod). 
+
+{{< tip >}}
+Note: Although the figure shows the HBONE tunnels to be between the two ztunnel proxies, in the current implementation with the inpod redirection model (from istio 1.21.0 onwards), the tunnels are in fact between the source and destination pods and traffic is HBONE encapsulated and encrypted in the network namespace of the source pod itself and eventually decapsulated and decrypted in the network namespace of the destination pod on the destination worker node. The ztunnel proxy still logically handles both the control plane and data plane needed for HBONE transport, however it is able to do that from inside the network namespaces of the source and destination pods. This is explained in more detail in the document [ztunnel datapath and inpod redirection](/docs/ops/ambient/usage/inpod).
```

Comment:
nit: should we update the diagrams and remove this explainer, or keep it? 

I am actually not sure.

Reply:
update diagram please